.travis.yml

```diff
@@ -31,3 +31,8 @@ script:
   - bundle exec rake
 
 jdk: oraclejdk8
+
+cache:
+  bundler: true
+  directories:
+    - .internal_test_app/tmp/cache/assets/test/sprockets
```

Comment:
So I'm a bit unfamiliar w/ how this works.  Will asset fingerprinting bust this cache?  (and maybe we can add a comment here, just so if we try to remove it again, we know what we're potentially getting into).

Thanks!